tox.ini

```diff
@@ -10,7 +10,7 @@ setenv =
 deps = -rdev-requirements.txt
 commands =
     /usr/bin/find . -name "*.pyc" -delete
-    flake8 {posargs}
```

Comment:
I removed this part because passing in any arguments to tox meant that they would either get passed to flake8 _or_  py.test. Since they have different arguments, this always led to one of them failing.